### PR TITLE
[MAINTENANCE] The 'sklearn' PyPI package is deprecated, use 'scikit-learn'

### PR DESCRIPTION
**Pull request by kenwade4**
Changes proposed in this pull request:
- Don't use deprecated PyPi package sklearn, only scikit-learn

### CODE REVIEW

File: great_expectations/core/usage_statistics/package_dependencies.py [hunk 1]
```diff
@@ -195,7 +195,6 @@ class GXDependencies:
         "scikit-learn",
         "shapely",
         "simple_icd_10",
-        "sklearn",
         "sympy",
```
Comment (Shinnnyshinshin) [inline great_expectations/core/usage_statistics/package_dependencies.py:198]: Just wanting to check : should `sklearn` here be replaced with `scikit-learn`?

File: reqs/requirements-dev-all-contrib-expectations.txt [hunk 2]
```diff
@@ -38,7 +38,6 @@ schwifty
 scikit-learn
 shapely
 simple_icd_10
-sklearn
```
Comment (Shinnnyshinshin) [inline reqs/requirements-dev-all-contrib-expectations.txt:41]: same question as above

File: great_expectations/core/usage_statistics/package_dependencies.py (hunk 1 above)
Comment (kenwade4) [reply great_expectations/core/usage_statistics/package_dependencies.py:198]: Yes, but scikit-learn is already in that list. Ditto for other comment.

**Review by Shinnnyshinshin — APPROVED** [review]
LGTM. Thanks for the clarification

**Pull request merged**